src/actions/actions.test.js

```diff
@@ -33,6 +33,16 @@ import {
   toggleTagActive,
   toggleTagFilter
 } from '../actions/tags';
+import {
+  TOGGLE_DISPLAY_CHONKY_GRAPH,
```

Comment:
Thanks @bru5 ! As per our discussion and final decision this morning, I had updated all references of the codebase to refer with the term 'large' instead of 'chonky'.